spec.html

```diff
@@ -47397,7 +47397,7 @@ <h1>Date.prototype.toGMTString ( )</h1>
         <emu-note>
           <p>The `toUTCString` method is preferred. The `toGMTString` method is provided principally for compatibility with old code.</p>
         </emu-note>
-        <p>The function object that is the initial value of `Date.prototype.toGMTString` is the same function object that is the initial value of `Date.prototype.toUTCString`.</p>
+        <p>The initial value of the `Date.prototype.toGMTString` property is %Date.prototype.toUTCString%, defined in <emu-xref href="#sec-date.prototype.toutcstring"></emu-xref>.</p>
```

Comment:
```suggestion
        <p>The initial value of the *"toGMTString"* property is %Date.prototype.toUTCString%, defined in <emu-xref href="#sec-date.prototype.toutcstring"></emu-xref>.</p>
```

Reply:
Whether to use the "full name" or just the property key is the same point that @michaelficarra brought up. Again, I'd like to defer that to a bigger PR (after some discussion about what's wanted).

Reply:
I agree in general, but in this specific case I think it makes sense to match the nearby sections right now (specifically the preceding [String.prototype.trimLeft](https://tc39.es/ecma262/#String.prototype.trimleft) and [String.prototype.trimRight](https://tc39.es/ecma262/#String.prototype.trimright), which this PR updates to «`The initial value of the *"trim{Left,Right}"* property is…`»).

Reply:
I agree with @gibson042. While it's true that there's lots of other inconsistency remaining to be fixed, it is also true that as of this PR there are 44 uses of precise phrase "\<p>The initial value of the X property", and of them only `Number.parseFloat`, `Number.parseInt`, `` %TypedArray%`.prototype.toString` ``, and `Date.prototype.toGMTString` are written out as full paths, and all of those uses are introduced by this PR. Seems best to just make those four consistent with the other uses of that phrase for the moment, and possibly change them all as part of #2576.

Reply:
Hm. Maybe I'll just pause this PR until #2576 makes some progress on question (1).

Reply:
Ehhh, I'd prefer to just land this, with the specific inconsistency mentioned above fixed, and then deal with #2576 later. I'm happy to push up a commit to this branch addressing the above comment if you'd prefer not to.

Reply:
Nah, you wore me down. I'll push a commit.